### ISSUE TITLE
Random bug (I think)

### ISSUE THREAD

**Issue by Kufato** [ael-bekk/PIPEX_TESTER#2]
Hi I wanted to clone your test to try it.
I followed the instructions well but I had a little problem when I try to launch it.
I have this error message that appears : ./tester.sh: 39: Syntax error: Bad for loop variable
I just wanted to know if I'm a bit stupid or just a little bug.
Thanks

**Comment by bastienkody** [comment]
yes i had the same. I just deleted all the lines that prints the MINISHELL artwork (line 32 to 62 included) and it worked

**Comment by ael-bekk** [comment]
I just tested it but I didn't see this msgs
